### DISCUSSION TITLE
Custom `TART_HOME` for gitlab runner?

### DISCUSSION THREAD

**Discussion by CatStudioApp** [cirruslabs/tart#738] [Q&A]
hello! is there a way to use custom `TART_HOME` for gitlab runner?

**Comment by edigaryev** [comment]
Hello @CatStudioApp, are you using [GitLab Tart Executor](https://github.com/cirruslabs/gitlab-tart-executor) or something else?

**Reply by CatStudioApp** [reply]
yes [GitLab Tart Executor](https://github.com/cirruslabs/gitlab-tart-executor)

**Reply by edigaryev** [reply]
Then you can just pass the `TART_HOME` environment variable directly to the GitLab Runner:

```shell
TART_HOME=/path/to/tart/home gitlab-runner run
```

Or, if you're running GitLab Runner via [launchd](https://www.launchd.info/), add an `EnvironmentVariables` key:

```xml
<key>EnvironmentVariables</key>
<dict>
<key>TART_HOME</key>
  <string>/path/to/tart/home</string>
</dict>
```
